libs/SmartSync/SmartSync/Classes/Manager/SFSmartSyncSyncManager.m

```diff
@@ -225,7 +215,7 @@ - (void) runSync:(SFSyncState*) sync updateBlock:(SFSyncSyncManagerUpdateBlock)u
 
     SyncFailBlock failSync = ^(NSString* message, NSError* error) {
         __strong typeof(weakSelf) strongSelf = weakSelf;
-        [strongSelf log:SFLogLevelError format:@"Sync type:%@ id:%d FAILED cause:%@ error:%@", [SFSyncState syncTypeToString:sync.type], sync.syncId, message, error];
+        LogSyncError(@"runSync failed:%@ cause:%@ error%@", sync, message, error);
```

Comment:
This is warning specifically on the name `message`, ultimately because it's falling down into the `SF_LOG_MAYBE()` definition, where `message` is also defined inline.  Suggest naming this variable to something else (e.g. `failureMessage`).